go/teams/resolve.go

```diff
@@ -178,7 +194,7 @@ func verifyResolveResult(ctx context.Context, g *libkb.GlobalContext, resolvedAs
 	}
 
 	if resolvedAssertion.ResolveResult.IsServerTrust() && g.Env.GetRunMode() != libkb.ProductionRunMode {
-		g.Log.CDebugf(ctx, "Trusting the server on assertion: %q (server trust - no way for clients to verify)", resolvedAssertion.Assertion.String())
+		g.Log.CWarningf(ctx, "Trusting the server on assertion: %q (server trust - no way for clients to verify)", resolvedAssertion.Assertion.String())
```

Comment:
This would mean users see log lines when attached to the CLI while the service is doing just about anything related to impteams with phone numbers wouldn't it?

Reply:
good point

Reply:
the idea was to see this warning when doing
```
 ~ » kb chat send "48784578263@phone" "hello"
▶ WARNING not exportable error: invalid team name: team names must be between 2 and 16 characters long (*errors.errorString)
Creating a new PRIVATE CHAT conversation
 ~ »
```

but this does not seem to be the case. Maybe let's just change it back to debug?